crates/zkvm-interface/src/lib.rs

```diff
@@ -17,7 +17,12 @@ pub trait Compiler {
     type Program: Clone + Send + Sync;
 
     /// Compiles the program and returns the program
-    fn compile(path_to_program: &Path) -> Result<Self::Program, Self::Error>;
+    ///
+    /// # Arguments
+    /// * `mount_directory` - The base directory (workspace root)
```

Comment:
`mount_directory` feels like a stretch now that only SP1 is the only one using Docker for guest program compiling. But I think at the end of the story, this sounds like the right name. Open to feedback!